string fileName
string text
---
string msgResp